pkg/oc/lib/newapp/cmd/newapp.go

```diff
@@ -170,6 +187,31 @@ func (e ErrRequiresExplicitAccess) Error() string {
 // ErrNoInputs is returned when no inputs are specified
 var ErrNoInputs = errors.New("no inputs provided")
 
+// TODO: we should run entire new-app using unstructured only
```

Comment:
@deads2k that's the nasty I've mentioned you yesterday, but I'm leaving a TODO here, when we have time we need to switch entire new-app to work unstructured only -> https://github.com/openshift/origin/issues/21983